default.env

```diff
@@ -166,19 +159,12 @@ JWT_SECRET=
 DISTRIBUTED=
 # Authenticated execution client endpoint. This default uses the execution node container.
 EL_NODE=http://execution:8551
-# EL node for Lido Validator Ejector
-VE_EXECUTION_NODE_URL=http://execution:8551
```

Comment:
I think we do need something here. Maybe call it `OBOL_EL_NODE` , and likewise have an `OBOL_CL_NODE`. Those can default to `http://execution:8545` and `http://consensus:5052` by default.

Background is the ability to run Lido CSM with CL and EL external to the Lido CSM Eth Docker stack.

Reply:
Yeah, it makes sense! Added